e2e-tests/fixtures/SequenceEditor.ts

```diff
@@ -0,0 +1,144 @@
+import { Locator, Page, expect } from '@playwright/test';
```

Comment:
Can this fixture file be renamed to `Sequencing`? `SequenceEditor` gives the impression that this just contains editor logic, but this fixture actually deals with managing sequences as well.

Reply:
I renamed the file :)